### PR TITLE
Remove unnecessary tooltip on device activity page

### LINKED ISSUE #26032: Duplicate tooltip/label for refresh button on device activity page
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

account/ui

### Describe the bug

In the account console v3, the refresh button on the `Account security` -> `Device activity` page has an unnecessary tooltip that exactly matches the label of the button.

### Version

nightly

### Expected behavior

The tooltip should either display a more verbose explanation of the button or not be displayed at all.

### Actual behavior

image

### How to Reproduce?

Open the account console v3 (enabled by default in nightly), navigate to `Account security` -> `Device activity`, and mouse over or tab to the "Refresh the page" button.

### Anything else?

Account console v2 uses the translation key `refresh` for the label and `refreshPage` for the tooltip:
https://github.com/keycloak/keycloak/blob/55d0a32b09a4e2d32f689c7b8ed0b864ca812854/themes/src/main/resources/theme/keycloak.v2/account/src/app/content/device-activity-page/DeviceActivityPage.tsx#L228-L238

Account console v3 uses `refreshPage` for both:
https://github.com/keycloak/keycloak/blob/55d0a32b09a4e2d32f689c7b8ed0b864ca812854/js/apps/account-ui/src/account-security/DeviceActivity.tsx#L118-L128

Using the shorter "Refresh" for the label probably makes sense, but the translation key would have to be copied from v2 to v3 (including localizations).

### PR DESCRIPTION
**Pull request by ReneZeidler**
Removes a tooltip that just repeats the same text as the label of the button itself.

Closes #26032

### CODE REVIEW

File: js/apps/account-ui/src/account-security/DeviceActivity.tsx [hunk 1]
```diff
@@ -115,17 +114,15 @@ const DeviceActivity = () => {
...
-              id="refresh-page"
-              variant="link"
-              onClick={() => refresh()}
-              icon={<SyncAltIcon />}
-            >
-              {t("refreshPage")}
-            </Button>
-          </Tooltip>
+          <Button
+            aria-describedby="refresh page"
```
Comment (jonkoops) [inline js/apps/account-ui/src/account-security/DeviceActivity.tsx:118]: I actually cannot find any element that has that has this `id`, and in this case the element is already descriptive, so let's remove this attribute entirely.
```suggestion
```

**Review by jonkoops — CHANGES_REQUESTED** [review]
Code looks good, but since we're touching this anyways, let's clean it up a little.

**Comment by ReneZeidler** [comment]
I actually noticed this earlier and forgot about it again.
Thanks, I've amended the PR. 😄

**Review by jonkoops — APPROVED** [review]
LGTM

**Comment by jonkoops** [comment]
Thanks for the fix @ReneZeidler!

**Pull request merged**